examples/browser/webpack.config.js

```diff
@@ -28,5 +34,15 @@ module.exports = require("../../config/webpack/webpack.config")(__dirname, {
         port: process.env.PORT
     }, plugins: [
         new webpack.HotModuleReplacementPlugin()
-    ]
-});
\ No newline at end of file
+    ],
+    resolve: {
+	// These shims are needed for bunyan
+	alias: {
+            'dtrace-provider': path.resolve(__dirname, './webpack_empty.js'),
+            fs: path.resolve(__dirname, './webpack_empty.js'),
```

Comment:
Humm could you elaborate ?

 I did not do much research on webpack yet... I just followed the bunyan doc, So I'm not sure about the interactions between  webpack.config.web fs:'empty' and the alias I have here...